atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -264,7 +265,9 @@ public static LockAndTimestampServices createLockAndTimestampServices(
             Environment env,
```

Comment:
Yep, added. At the moment it is still used by the CLIs (atlasdb-dagger) but agree that it shouldn't be public.